### PR TITLE
VEN-39 | Remove styled-component 

### PR DESCRIPTION
**Pull request by midori0507**
- Remove styled-component completely
- Add Sass structure
- CSS structure follow BEM syntax.
- Move files into folders

===============
Component example
```
/[component]
      _styles.scss
      types.ts
      index.tsx
```

### CODE REVIEW

File: src/components/steps/Step/_styles.scss [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    &.current,
+    &.completed {
+      background-color: $hel-coat;
+    }
+  }
+
+  &__label {
+    font-weight: 500;
+    font-size: 0.66rem;
```
Comment (Nizar-Rahme) [inline src/components/steps/Step/_styles.scss]: Don't we have variables for font sizes?

File: src/components/forms/SectionSelector.tsx [hunk 2]
```diff
@@ -32,50 +27,30 @@ interface FormSelectWrapperProps {
...
-  line-height: 1;
-  ${responsive.sm`
-    font-size: 1em;
-    font-weight: ${(props: FormSelectWrapperProps) => (props.selected ? '600' : '400')};
-  `}
-`;
-
 const SectionSelector = ({ name, selected, types, sizes }: Props) => (
-  <ButtonWrapper>
+  <div className="app-Form__section-selector">
```
Comment (Nizar-Rahme) [inline src/components/forms/SectionSelector.tsx]: For the sake of consistency, how about `app-SectionSelector`?

File: src/components/forms/SectionSelector.tsx [hunk 3]
```diff
@@ -32,50 +27,30 @@ interface FormSelectWrapperProps {
...
     <Container>
       <Row>
         <Col lg={{ size: 10, offset: 1 }} xl={{ size: 8, offset: 2 }}>
           <Row>
             {types.map(({ label, tab, icon }: TypeProps) => (
               <Col id={`${tab}_selection`} key={`${name}.${tab}`} {...sizes}>
-                <FormSelectWrapper to={`form/${tab}`} selected={selected === tab}>
+                <LocalizedLink
+                  className={classNames('app-section-selector__link', {
+                    'is-selected': selected === tab
```
Comment (Nizar-Rahme) [inline src/components/forms/SectionSelector.tsx]: Following BEM, this `className` should be added as a modifier e.g: `app-SectionSelector__link--selected`

File: src/components/berths/index.tsx [hunk 4]
```diff
@@ -1,49 +1,27 @@
...
-  margin: 1em 0;
-`;
+import { BerthProps } from './types';
 
 const { REACT_APP_MAX_SELECTED_BERTHS } = process.env;
 
-export default ({ filtered, filteredNot, onClick, selected }: Props) => (
-  <Wrapper>
+export default ({ filtered, filteredNot, onClick, selected }: BerthProps) => (
+  <Container className="app-berths">
```
Comment (Nizar-Rahme) [inline src/components/berths/index.tsx]: Component names are sometimes capitalized and sometimes not, I prefer to stick to one or the other.

File: src/components/steps/Step/_styles.scss (hunk 1 above)
Comment (midori0507) [reply src/components/steps/Step/_styles.scss]: Great find, gonna fix, we dont but we should define

File: src/components/forms/SectionSelector.tsx (hunk 2 above)
Comment (midori0507) [reply src/components/forms/SectionSelector.tsx]: Sure.

File: src/components/forms/SectionSelector.tsx (hunk 3 above)
Comment (midori0507) [reply src/components/forms/SectionSelector.tsx]: Agree, my typo

File: src/components/berths/index.tsx (hunk 4 above)
Comment (midori0507) [reply src/components/berths/index.tsx]: capital it is, will fix

**Pull request merged**